marketplace_builder/views/pages/tutorials/performance/measuring-execution-time-liquid-time-diff.liquid

```diff
@@ -0,0 +1,94 @@
+---
+converter: markdown
+metadata:
+  title: Measuring Execution Time of Liquid Code Fragments (time_diff) 
+  description: This guide will help you measure the execution time of Liquid code fragments using the time_diff filter. 
+slug: /tutorials/performance/measuring-execution-time-liquid-time-diff
+searchable: true
+---
+
+This guide will help you measure the execution time of Liquid code fragments using the `time_diff` filter. You can use this quick and simple method to find out what is wrong when a page is slow. 
```

Comment:
This method doesnt tell you whats wrong when page is slow - it only tells which part of it is slow.

Reply:
Very good point, thank you!